components/apollo_io/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/apollo_io",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

New components being added
```suggestion
  "version": "0.3.0",
```